pkg/movies/handlers_test.go

```diff
@@ -29,6 +30,31 @@ func TestListHandler(t *testing.T) {
 		require.NoError(tt, err)
 		assert.True(tt, len(response) >= 23)
 	})
+
+	t.Run("Test limit and offset parameter", func(tt *testing.T) {
+		offset := 2
+		byteStr := fmt.Sprintf(`{"limit": 3, "offset": %d}`, offset)
```

Comment:
Is there a reason why you're hardcoding the limit but storing the offset value as a variable?

Reply:
Ah yes - I had added an assert call that used to refer to the offset variable and then do a hard check on whether or not that name was "captain marvel" or something like that. But then I changed that assertion test and forgot to change the variable back to a hard code and not a variable